apps/aggiemap-angular-e2e/src/integration/features-layers.spec.ts

```diff
@@ -18,75 +26,222 @@ desktopSizes.forEach((size) => {
 
     it('Construction Zone', function() {
       cy.wait('@construction')
-      cy.checkLayer('1', 'Construction Zone')
-      cy.checkLegend('9', 'Construction Area')
-      cy.checkIcon('9',this.data.construction, 'Construction Area')
+      cy.checkLayer('8', 'Construction Zone')
+      cy.checkLegend('8', 'Construction Zone')
+      // click location of a known construction zone for multiple resolutions
+      if (size[0] === 1920) {
+        cy.wait(2000)
+        cy.get('canvas').trigger('mouseover').click(900, 650)
+      }
+      else if (size[0] === 1366) {
+        cy.wait(2000)
+        cy.get('canvas').trigger('mouseover').click(625, 500)
+      }
+      else if (size[0] === 1440) {
+        cy.wait(2000)
+        cy.get('canvas').trigger('mouseover').click(670, 555)
+      }
```

Comment:
This is subject to breaking if the features on the screen coordinates are removed. For construction shapes, this happens quite often. I don't know a better way off the top of my head but try and look into accessing component state through cypress. If you can do that, you can maybe grab the map instance and query the screen and retrieve the screen coordinates of a shape that is rendered within the viewport.